example/src/components/network-graph/index.tsx

```diff
@@ -57,6 +57,7 @@ export const MyNetworkGraph = () => {
         nodeDistance={1000}
         LinkComponent={MyLine}
         NodeComponent={MyNode}
+        hoverOpacity={0.1}
```

Comment:
I think we need to remove this one? or pass it as a prop (in a new PR), so it is not always a fixed value

Reply:
Removed